source/puppet/5.1/config_about_settings.markdown

```diff
@@ -84,3 +84,95 @@ If a setting isn't specified on the command line or in `puppet.conf`, it falls b
 
 Some default values are based on other settings --- when this is the case, the default is shown using the other setting as a variable (similar to `$ssldir/certs`).
 
+{:.concept}
+## Configuring locale settings
+
+Puppet 5.2.0 added support for locale-specific strings in output, and it detects your locale from your system configuration. This provides localized strings, report messages, and log messages for the locale's language when available.
```

Comment:
Says 5.2.0.

Reply:
acknowledged but will fix post-merge, this is in the 5.1 docs and should not be a ship stopper